website/docs/essays/tx-pool-rbf.md

```diff
@@ -37,7 +45,7 @@ There are several check rule for RBF:
 
 5. The descendants of old transactions can not contains any transaction which is a ancestor of the new transaction.
```

Comment:
5. . The descendants of old transactions must not contain any transaction that is a ancestor of the new transaction.